src/backend/api/Fusion.Resources.Api/Notifications/InternalRequestNotification/NotifyResourceOwnerHandler.cs

```diff
@@ -26,7 +26,7 @@ protected override async Task Handle(NotifyResourceOwner request, CancellationTo
             {
                 var recipients = await GenerateRecipientsAsync(request.Editor.Person.AzureUniqueId, request.AssignedDepartment);
                 
-                var arguments = new NotificationArguments($"A personnel request has been assigned to you") { AppKey = "personnel-allocation" };
+                var arguments = new NotificationArguments($"A personnel request has been assigned to you (you are notified as resource owner)") { AppKey = "personnel-allocation" };
```

Comment:
Suggested text: You have been assigned as resource owner for a personnel request